src/model/advertisement.ts

```diff
@@ -19,7 +120,17 @@ export const namedAdSlotParameters = (name: AdSlotType): AdSlotParameters => {
             name: 'top-above-nav',
             adTypes: ['mpu-banner-ad', 'rendered'],
             sizeMapping: {
-                tablet: ['1,1|2,2|728,90|88,71|fluid'],
+                // mirror frontend file mark: 432b3a46-90c1-4573-90d3-2400b51af8d0
```

Comment:
what is a file mark??

Reply:
Ids that I use to refers to files or parts of a file from other places (in the same project or other projects) :)